alicloud/service_alicloud_common_test.go

```diff
@@ -105,3 +105,19 @@ resource "alicloud_vswitch" "default" {
   name              = "${var.name}"
 }
 `
+
+const ElasticsearchInstanceCommonTestCase = `
+data "alicloud_zones" "default" {
+    available_resource_creation = "VSwitch"
+}
+resource "alicloud_vpc" "default" {
+  name       = "${var.name}"
+  cidr_block = "172.16.0.0/16"
+}
+resource "alicloud_vswitch" "default" {
+  vpc_id            = "${alicloud_vpc.default.id}"
+  cidr_block        = "172.16.0.0/24"
+  availability_zone = "${data.alicloud_zones.default.zones.1.id}"
```

Comment:
I think the line  `availability_zone = "${data.alicloud_zones.default.zones.1.id}"` should be ` availability_zone = "${data.alicloud_zones.default.zones.0.id}"`